Medicine.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <string>
+
+using namespace std;
+
+class Medicine
+{
+private:
+	float price;
+	int quantity;
+	string name;
+	bool isPrescriptionNeeded;
+	string date;
+public:
+	Medicine() {
+		input();
+	}
+	Medicine(float price, int quantity, string name, bool t, string date) {
+		setPrice(price);
+		setQuantity(quantity);
+		setName(name);
+		setIsPrescriptionNeeded(t);
+		setDate(date);
+	}
+	void setPrice(float price) { this->price = price; }
+	void setQuantity(int quantity) { this->quantity = quantity; }
+		 
+	void setName(string name) { this->name = name; }
+	void setIsPrescriptionNeeded(bool t) { this->isPrescriptionNeeded = t; }
+	void setDate(string date) { this->date = date; }
+	float getPrice() { return price; }
+	int getQuantity() { return quantity; }
+	string getName() { return name; }
```

Comment:
Don't put method realization in .h file, put it to .cpp file even when there is one line. And method's names shouldn't be with uppercase, instead this use _

Reply:
I transferred the method get_name from file .h to .cpp file and put _